deployment/gke/first_time_setup.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
```

Comment:
Is this same as one_time_setup.sh?

Reply:
Yes, I put each sub-tasks in a separate script since need to re-run some of them independently (like creating new cluster)